### ISSUE TITLE
merge work keys and facet keys?

### ISSUE THREAD

**Issue by malantonio** [LafayetteCollegeLibraries/lafayette-preserve#31]
Currently, the doc-level responses contain one key and the facet-level response contains a solr-ized version of the same. (For example: `creator` in a work is found as `creator_sim` in the facet.) As there isn't a reference to this (that I can tell) in the `/concern/generic_works/:id.json` response, any links to a faceted search return no responses.

Is it possible to merge these fields so that a key returned in a Work response would correspond to a facet field?

**Comment by jrgriffiniii** [comment]
> Is it possible to merge these fields so that a key returned in a Work response would correspond to a facet field?

As I understand this, you want `/concern/generic_works/:id.json` response bodies to contain keys structured as Solrized field names rather than attribute names on the `CurationConcern::Work` Objects?

**Comment by malantonio** [comment]
Yes please! As far as I can tell, that's the only way to build out faceted links from within a Work without building a dictionary of `work_field => solrized_name`.
